Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12-alpine
+FROM golang:1.16-alpine
 ENV GO111MODULE=on
```

Comment:
This configuration setting is the default in [Go 1.16](https://golang.org/doc/go1.16#go-command). We should be able to remove this line.

Reply:
Updated. Thanks!